news/_posts/2024-11-01-playtest-20241101.md

```diff
@@ -0,0 +1,27 @@
+---
+kind: article
+title: "Playtest 20241101"
+author: "PunkPun"
+created_at: 2024-11-01 22:00 +0000
+disqus_id: "playtest-20241101"
+permalink: "/news/playtest-20241101/"
+---
+
+Welcome back, Commanders! After a whole year of development the OpenRA team proudly presents [Playtest 20241101]({{ '/download' | relative_url }}).
+
+This update brings a large array of changes. Most notably, speed! Whatever you do, you should feel a significant difference from last release, especially when playing against AI!
+
+We present a new map editor! It has been revamped to feel more modern and have a more uniform design. Added features include: draggable selection, measurement tools, marker layer, play button, draggable actors and more!
+
+Some miscellaneous features include:
+- Added support for installing game assets from the Steam release of The Ultimate Collection.
+- Added two Red Alert missions, and improved the quality of many others.
+- Significantly improved support for HD art assets.
+- Switched to high-quality Dune 2000 assets and loads of extra visual polish.
+- Red Alert and Dune 2000 balance changes.
+- Added Tiberian Dawn EVA Database.
+- Persistent lobby options.
+
+While we made significant progress towards language localization, we are not yet ready to accept language contributions. Modders wanting to update, will be delighted to hear that we provide a number of tools for updating to the new fluent framework, as well as scrupulous lint tests.
```

Comment:
IMO this can be phrased more positively: "Many changes behind the scenes made significant progress towards supporting additional languages in a future release"